packages/webpack-config-single-spa/lib/webpack-config-single-spa.js

```diff
@@ -120,6 +121,15 @@ function webpackConfigSingleSpa(opts) {
         },
       },
       allowedHosts: "all",
+      hot: outputSystemJS,
```

Comment:
The diff for this pull request looks wrong. The main branch already has this line, and also has `const outputSystemJS = !!opts.outputSystemJS`, but github's diff isn't showing that. Do you know what is causing the diff to be wrong? 

See related:
https://github.com/single-spa/create-single-spa/blob/88364e898189e899050a6e3f90389367a39e9dcf/packages/webpack-config-single-spa/lib/webpack-config-single-spa.js#L42

Reply:
prbly this: husky hook did something i didn't expect at `git add -p && git commit`, didn't notice till opening PR and then amended. I'll merge upstream and diff should be right